### DISCUSSION TITLE
Newbie question - nvim-tree open file issue

### DISCUSSION THREAD

**Discussion by aokojshdgop8791** [LunarVim/LunarVim#4323] [Q&A] [answered]
Just installed lunarvim 1.3 with neovim 0.9 installed.  When I open a file, nvim-tree will expand the folders all the way to my home directory, is there a way to disable that behaviour?  Thank you

**Comment by LostInTheLogs** [comment, marked as answer]
duplicate of https://github.com/LunarVim/LunarVim/issues/3362
